muskathlon/models/event_compassion.py

```diff
@@ -46,3 +110,24 @@ class MuskathlonRegistration(models.Model):
         ('reg_unique', 'unique(event_id,partner_id)',
          'Only one registration per participant/event is allowed!')
     ]
+
+    def _compute_amount_raised(self):
+        muskathlon_report = self.env['muskathlon.report']
+
+        for registration in self:
+            amount_raised = int(sum(
+                item.amount for item in muskathlon_report.search([]) if
+                item.user_id.id == registration.partner_id.id))
```

Comment:
why not putting this condition in the search method? This would be more efficient.

Reply:
Yes, I've done it.